build.sbt

```diff
@@ -3763,7 +3763,7 @@ lazy val `engine-runner` = project
     Compile / run / mainClass := Some("org.enso.runner.Main"),
     commands += WithDebugCommand.withDebug,
     inConfig(Compile)(truffleRunOptionsSettings),
-    libraryDependencies ++= GraalVM.modules ++ jline ++ Seq(
+    libraryDependencies ++= GraalVM.modules ++ GraalVM.toolsPkgs ++ jline ++ Seq(
```

Comment:
What is the impact of `GraalVM.toolsPkgs` on NI size?

Reply:
Good question.
- with 4ca442256fb0c005cc8c83a981cddb5bddb54841 
- the size of `ENSO_LAUNCHER=native sbt buildEngineDistribution`
- on Linux is `426969352`